### ISSUE TITLE
a11y - site setup - Content Types - sidebar "add new content type" button  - cms ui

### ISSUE THREAD

**Issue by Wagner3UB** [plone/volto#5141]
**Describe the bug**
- The icon "+" (add new content type) inside de left sidebar doesn't receive a visible focus when select, either on hover or with the keyboard. After the click, a modal/pop-up appears, but the focus remains on the "+" button and it does not enter the modal.

**To Reproduce**

1. Go to "site setup" -> "Content types"
2. With a screen reader + keyboard, go inside the left toolbar on the + icon.
3. press enter to access the next behavior
4. a pop-up/modal will appear

**Expected behavior**
- The icon must have a visible focus, either on hover or with the keyboard, and the modal/pop-up must receive the focus after the keypress on the "+" button.
- The focus must remain in a loop inside the modal until the save or the cancel button is pressed.

**Screenshots**
<img width="1568" alt="Schermata 2023-08-28 alle 17 07 44" src="https://github.com/plone/volto/assets/60133113/204162b3-c434-4b66-9756-7e16d6c2ecbe">
<img width="1513" alt="Schermata 2023-08-28 alle 17 07 52" src="https://github.com/plone/volto/assets/60133113/63c7e9e9-f307-4af8-a398-4093e2c2aa51">

**Software (please complete the following information):
Volto Version 17.0.0-alpha.19
Plone Version 6.0.4

**Comment by MayaSatishRao** [comment]
@Wagner3UB i want to work on this issue? can you assign this to me?

**Comment by stevepiercy** [comment]
@MayaSatishRao please follow our contributing guidelines:

- https://6.docs.plone.org/contributing/index.html
- https://6.docs.plone.org/contributing/first-time.html

**Comment by ShatilKhan** [comment]
Hi @Wagner3UB 
Can you assign me this issue?
I have worked on frontend contributions before.

**Comment by Wagner3UB** [comment]
@ShatilKhan please follow https://6.docs.plone.org/contributing/index.html and https://6.docs.plone.org/contributing/first-time.html

**Comment by Noobham** [comment]
Hi @Wagner3UB 

I am working on this issue. The goal is to set aria-hidden="false" on the root element and aria-hidden="true" on modal when a popup is displayed. When the popup is closed, these attributes should be toggled back to their original states.

facing issue in finding root element of Volto ?
Also can i know is my idea correct or should i find any other alternative.
